Cargo.toml

```diff
@@ -31,9 +31,26 @@ rand_core = "0.5.1"
 zcash_history = "0.2"
 zcash_primitives = "0.2"
 zcash_proofs = "0.2"
+zcash_extensions = "0.0"
 ed25519-zebra = "2.0.0"
 
 [profile.release]
 lto = true
 panic = 'abort'
 codegen-units = 1
+debug = true
+opt-level = 0
+
+[profile.dev]
+debug = true
+opt-level = 0
+
+[patch.crates-io]
+bellman = {git = "https://github.com/nuttycom/librustzcash", branch = "zip-tzes"}
```

Comment:
Moving from versioned dependencies to git dependencies for our main Rust libraries is not ideal. I would prefer that we merge the Rust changes and then make `v0.3.0-beta.1` releases (given that we won't be ready to make the actual `v0.3.0` releases until the last few refactoring PRs are merged).

In any case, this currently breaks compilation because the depends system is not vendoring the updated dependencies. They would need to be regenerated similarly to https://github.com/zcash/zcash/pull/4581/commits/9f71fa1c09692ae985ac49f29770fee62d0b0b99 (or just with the `vendorcrate.sh` script once using published crate versions).

Reply:
@str4d let me know once we publish an updated librustzcash release and I will remove this section.